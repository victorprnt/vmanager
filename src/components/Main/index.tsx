import CardsContainer from '../CardsContainer'

import SideBar from '../SideBar'
import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    {/* <SideBar aria-label="Side Bar" /> */}
    <CardsContainer />
  </S.Wrapper>
)

export default Main
