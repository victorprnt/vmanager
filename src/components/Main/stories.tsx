// import { Story, Meta } from '@storybook/react'
import Main from '.'

// export default {
//   title: 'Main',
//   component: Main
// } as Meta

// export const Default: Story = () => <Main />
