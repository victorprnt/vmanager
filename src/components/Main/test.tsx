import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should render the main page', () => {
    const { container } = render(<Main />)

    expect(screen.getByRole('navigation')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
