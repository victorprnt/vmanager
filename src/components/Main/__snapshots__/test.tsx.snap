// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the main page 1`] = `
<div
  class="sc-hKMtZM hOqbbF"
>
  <section
    class="sc-dkzDqf grYqif"
  >
    <div
      class="infocard-container"
    >
      <div
        class="sc-gsnTZi ggXPc infocard"
      >
        <img
          alt="Infocard Image"
          class="infocard"
          src=""
        />
        <div
          class="infocard-text"
        >
          <p
            class="infocard-value"
          >
            Value
          </p>
          <p
            class="infocard-label"
          >
            Label
          </p>
        </div>
      </div>
      <div
        class="sc-gsnTZi ggXPc infocard"
      >
        <img
          alt="Infocard Image"
          class="infocard"
          src=""
        />
        <div
          class="infocard-text"
        >
          <p
            class="infocard-value"
          >
            Value
          </p>
          <p
            class="infocard-label"
          >
            Label
          </p>
        </div>
      </div>
      <div
        class="sc-gsnTZi ggXPc infocard"
      >
        <img
          alt="Infocard Image"
          class="infocard"
          src=""
        />
        <div
          class="infocard-text"
        >
          <p
            class="infocard-value"
          >
            Value
          </p>
          <p
            class="infocard-label"
          >
            Label
          </p>
        </div>
      </div>
    </div>
    <div
      class="sc-bczRLJ iZVTKG"
    >
      <h2
        class="graph-title graph-text"
      >
        Graph Title
      </h2>
      <h3
        class="graph-subtitle graph-text"
      >
        subtitle
      </h3>
      <img
        alt="Graph image"
        src=""
      />
    </div>
    <div
      class="sc-bczRLJ iZVTKG"
    >
      <h2
        class="graph-title graph-text"
      >
        Graph Title
      </h2>
      <h3
        class="graph-subtitle graph-text"
      >
        subtitle
      </h3>
      <img
        alt="Graph image"
        src=""
      />
    </div>
  </section>
</div>
`;
